src/app/components/panel-combat-combatlog/panel-combat-combatlog.component.ts

```diff
@@ -10,10 +11,27 @@ import { combatLog } from '../../helpers';
   styleUrl: './panel-combat-combatlog.component.scss',
 })
 export class PanelCombatCombatlogComponent {
+  private createCustomRenderer() {
+    const renderer = new marked.Renderer();
+
+    renderer.codespan = ({ text }: { text: string }) => {
+      if (text.startsWith('rarity:')) {
+        const [_, rarity, itemName] = text.split(':');
```

Comment:
I think, to address the linter, this can be `const [, rarity, itemName] = text.split(':')` but I'm not sure offhand if that compiles. It should, though.